### DISCUSSION TITLE
Show function parameters in auto completion list

### DISCUSSION THREAD

**Discussion by selimaj-dev** [microsoft/pylance-release#4725] [Enhancement]
Ok so i tried Pycharm and i didn't like it but i loved the autoguestions menu

[Screen.Recording.2023-03-07.at.12.02.49.PM.mov ](https://user-images.githubusercontent.com/103524696/223404693-eda11cf7-2d9e-4fcd-86f3-966fe0f48125.mov)

i want the arguments to be at the function like:

print(values, sep, end, file flush)

instead of:

print

![](https://user-images.githubusercontent.com/103524696/223407214-d97000de-9e3e-43e8-aa1b-7c987b76d527.png)

**Comment by bschnurr** [comment]
Thanks for the great suggestion. Moving to discussions for upvotes.
